cmd/istioctl/main.go

```diff
@@ -318,10 +346,12 @@ Example usage:
 func init() {
 	rootCmd.PersistentFlags().StringVarP(&kubeconfig, "kubeconfig", "c", "",
 		"Use a Kubernetes configuration file instead of in-cluster configuration")
-	rootCmd.PersistentFlags().StringVarP(&managerAddr, "managerAddr", "m", "",
-		"Set your Istio manager address")
 	rootCmd.PersistentFlags().StringVarP(&namespace, "namespace", "n", api.NamespaceDefault,
 		"Select a Kubernetes namespace")
+	rootCmd.PersistentFlags().StringVar(&istioNamespace, "istioNamespace", defaultIstioNamespace,
+		"Namespace where Istio system resides")
+	rootCmd.PersistentFlags().IntVar(&apiServerPort, "port", 8081,
+		"Manager API server port")
 
```

Comment:
I don't think there is any disagreement that istioctl should be platform neutral long-term. But I wanted to minimize changes to istioctl *and* the rest of the Istio install prior to alpha freeze. In particular, accessing kube API server via JWT tokens still requires k8s specific calls to get the token, certs, and API server address vs. using the existing k8s client to do that all for us.   

I added the `--kube` flag to revert to the legacy behavior. Advanced users will need to change the `--managerAPIService` to localhost:<PORT> and run `kubectl port-forward` themselves as before. 

```bash
iostioctl --kube=false --managerAPIService=http://localhost:8081 create -f route-rule-reviews-v2-v3.yaml
```

The default behavior is `--kube=true`.

```bash
iostioctl create -f route-rule-reviews-v2-v3.yaml
```